framework-api-event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/EventSource.java

```diff
@@ -13,32 +13,39 @@
     /**
      * Get a stream of events by stream id.
```

Comment:
Is this getting a stream of events, or is it getting an eventstream? getStreams() method gets the actual events